src/main/kotlin/frc/robot/subsystems/intake/IntakeIO.kt

```diff
@@ -0,0 +1,41 @@
+package frc.robot.subsystems.intake
+
+import edu.wpi.first.units.Angle
+import edu.wpi.first.units.Measure
+import edu.wpi.first.units.Units
+import org.team9432.annotation.Logged
+
+interface IntakeIO {
+    val inputs: LoggedIntakeInputs
+
+    fun setSpinPower(power: Double) {
+
+    }
+
+    fun setCenterPower(power: Double) {
+
+    }
+
+    fun setAngle(angle: Measure<Angle>) {
+
+    }
+
+    fun setAnglePower(power: Double) {
+
+    }
+
+    fun reset() {
```

Comment:
This name is too general, I had to look at the implementation to understand the meaning. `retract` or `fold` are clearer.

Reply:
It doesn't retract the intake, it resets the encoder.